File: 8kyu/(8 kyu) Twice as old/(8 kyu) Twice as old.js
// 1 Plain solution
function twiceAsOld(dadYearsOld, sonYearsOld) {
    const twiceOld = sonYearsOld * 2;
    const res = dadYearsOld - twiceOld;
    if (res > 0) {
        return res;
    } else {
        return -res;
    }
}

// 2 Optimized solution
function twiceAsOld(dadYearsOld, sonYearsOld) {
    return Math.abs(dadYearsOld - sonYearsOld * 2);
}

// 3 Coding golf
const twiceAsOld = (d, s) => Math.abs(d - s * 2);

File: 8kyu/(8 kyu) Twice as old/(8 kyu) Twice as old.cs
namespace Solution
{
    public class TwiceAsOldSolution
    {
        public static int TwiceAsOld(int dadYears, int sonYears)
        {
            int twiceOld = sonYears * 2;
            int res = dadYears - twiceOld;
            if (res > 0)
            {
                return res;
            }
            return -res;
        }
    }
}